bin/cli.js

```diff
@@ -3,17 +3,17 @@
 'use strict';
 
 const minimist = require('minimist');
-const _ = require('lodash');
+const camelCase = require('lodash.camelcase');
 const checkDependencies = require('../lib/check-dependencies');
 
 const argv = minimist(process.argv.slice(2));
 
 // camelCase the options
-for (const key in argv) {
+Object.keys(argv).forEach(key => {
```

Comment:
I guess my main point is that for simple objects where neither performance nor complex prototype chain handling matters `for (const key in argv) {` reads nicer than the `Object.keys(argv).forEach(key => {` counterpart.

There's a third option as well:
```js
for (const key of Object.keys(argv)) {
    /* ... */
}
```

If we're talking performance, it should be even faster. ;)